src/devices/bus/isa/fdc.h

```diff
@@ -83,11 +84,25 @@ class isa8_fdc_superio_device : public isa8_fdc_device {
 	virtual void device_add_mconfig(machine_config &config) override;
 };
 
+class isa8_ec1841_0003_device : public isa8_fdc_device {
+public:
+	isa8_ec1841_0003_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+
+	DECLARE_WRITE_LINE_MEMBER( aux_irq_w );
```

Comment:
This should be protected since it's an internal signal handler.  There's no need to make handlers used from machine configuration public any more.